bw2io/export/excel.py

```diff
@@ -11,9 +11,27 @@
 
 
 def create_valid_worksheet_name(string):
-    """Exclude invalid characters and names.
+    # """Exclude invalid characters and names.
```

Comment:
...same question here: [why are you keeping the old docstrings in and commenting them out?](https://github.com/brightway-lca/brightway2-io/pull/169/files#r1141269894)